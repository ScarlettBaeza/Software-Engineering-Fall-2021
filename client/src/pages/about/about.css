@import url(https://fonts.googleapis.com/css?family=Roboto:400,300,500);
  
h1 {
    margin: 0 0 20px 0;
    font-weight: 300;
    font-size: 28px;
    color: white;
}
