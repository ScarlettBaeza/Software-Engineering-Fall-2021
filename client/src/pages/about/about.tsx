import React from 'react';
import './about.css';

export const About = () => {
    return (
        <>
        <div className="aboutSection">
            <h1>About Us</h1>
            <p>This project was created for COSC 4351 at the University of Houston.</p>
            <h1>Creators</h1>
            <h1>Scarlett Baeza, Eric Nguyen, Carlos Rosales</h1>
        </div>
        
        </>
    );    
}