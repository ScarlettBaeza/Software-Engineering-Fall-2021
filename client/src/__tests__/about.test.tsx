import React from 'react';
import renderer from 'react-test-renderer';
import { About} from '../pages/about/about';
import '@testing-library/jest-dom';

it('about page renders correctly', () => {
    const tree = renderer
      .create(<About/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });