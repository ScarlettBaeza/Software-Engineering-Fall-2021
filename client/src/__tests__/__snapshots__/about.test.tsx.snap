// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about page renders correctly 1`] = `
<div
  className="aboutSection"
>
  <h1>
    About Us
  </h1>
  <p>
    This project was created for COSC 4351 at the University of Houston.
  </p>
  <h1>
    Creators
  </h1>
  <h1>
    Scarlett Baeza, Eric Nguyen, Carlos Rosales
  </h1>
</div>
`;
